src/utils/chains.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * List of all the networks supported by the Uniswap Interface
+ */
+export enum SupportedChainId {
```

Comment:
You can delete this file, should be handled in #4

Reply:
Deleted